test_util/src/servers/ws.rs

```diff
@@ -2,26 +2,25 @@
 
 use anyhow::anyhow;
 use bytes::Bytes;
-use fastwebsockets::FragmentCollector;
-use fastwebsockets::Frame;
-use fastwebsockets::OpCode;
-use fastwebsockets::Role;
-use fastwebsockets::WebSocket;
+use fastwebsockets_06::FragmentCollector;
+use fastwebsockets_06::Frame;
+use fastwebsockets_06::OpCode;
+use fastwebsockets_06::Role;
+use fastwebsockets_06::WebSocket;
 use futures::future::join3;
 use futures::future::poll_fn;
 use futures::Future;
 use futures::StreamExt;
-use h2::server::Handshake;
-use h2::server::SendResponse;
-use h2::Reason;
-use h2::RecvStream;
-use hyper::service::service_fn;
-use hyper::upgrade::Upgraded;
-use hyper::Body;
-use hyper::Method;
-use hyper::Request;
-use hyper::Response;
-use hyper::StatusCode;
+use h2_04::server::Handshake;
+use h2_04::server::SendResponse;
+use h2_04::Reason;
+use h2_04::RecvStream;
```

Comment:
This is ugly, but will be cleaned up and once https://github.com/denoland/deno/issues/21578 is done.